components/Modal.jsx

```diff
@@ -0,0 +1,34 @@
+import { useRef, useEffect } from "react";
+import styles from "@/styles/components/modal.module.scss";
+
+export default function Modal({ content, isModalOpen, setIsModalOpen }) {
```

Comment:
`isModalOpen` is redundant as you are already in Modal context. Call it `isOpen` instead. The same with the setter.

Reply:
noted